recipes/mpiplus/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "mpiplus" %}
+{% set version = "v0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/choderalab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a3d50157a274decab87589ad78b9e41e9f3d34eb27a7d50cc6e7ef34def98b44 
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - coverage>=4.4
```

Comment:
I don't think that coverage is needed in host, since it is a testing tool (and one that doesn't make a lot of sense for a normal package.)